backend/src/main/java/com/hufudb/onedb/backend/controller/UserController.java

```diff
@@ -87,6 +100,24 @@ void dropGlobalTable(@PathVariable String name) {
 
   @PostMapping("/user/query")
   PojoResultSet query(@RequestBody Request request) {
-    return PojoResultSet.fromResultSet(onedb.executeQuery(request.value));
+    Long id;
+    LocalDateTime submit = LocalDateTime.now();                           // get submit Time; Status = Submitted
+    synchronized (this) {
+      if (flag) {
+        maxId = sqlRecordService.getMaxId();
+        flag = false;
+      }
+      maxId = maxId + 1;
+      id = maxId;
+    }
+    sqlRecordService.submit(id, request.value, "qlh", submit);    // store into sqlRecord database
+    LocalDateTime start = LocalDateTime.now();                             // get start time
```

Comment:
When defining the ddl of table schema, we can give the column with default value current_timestamp to avoid assign value through code, in which case the time of different server will be different.